tests/restarting/to_7.3.0/CycleTestRestart-2.toml

```diff
@@ -2,6 +2,11 @@
 maxTLogVersion = 6
 disableTss = true
 
+[[knobs]]
+# testing that restart with writing recover at version to CSTATE.
+record_recover_at_in_cstate = true
```

Comment:
Will this knob be enabled in 7.3? If not, then we need to test when this is false.

Reply:
discussed offline.